forte/ontology_specs/ag_news.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "ag_news",
+    "imports": [
+      "base_ontology.json"
+    ],
+    "definitions": [
+      {
+        "entry_name": "ft.onto.ag_news.Article",
+        "parent_entry": "ft.onto.base_ontology.Document",
+        "attributes": [
+            {
+              "name": "class_id",
+              "description": "The class ids are numbered 1-4 where 1 represents World, 2 represents Sports, 3 represents Business and 4 represents Sci/Tech.",
+              "type": "int"
+            },
+            {
+                "name": "class_name",
+                "description": "The name of the class.",
```

Comment:
The description is not informative enough, this basically provides the same amount of information as the `name` itself.